components/eamxx/src/share/io/scorpio_output.hpp

```diff
@@ -11,7 +11,10 @@
 
 #include "ekat/ekat_parameter_list.hpp"
 #include "ekat/mpi/ekat_comm.hpp"
+#include "ekat/ekat_pack_utils.hpp"
+#include "ekat/ekat_pack_kokkos.hpp"
 
+#include <numeric>
```

Comment:
I am not sure these three new headers are used in this file. If you need them in the cpp file, you can include them there. But I don't think you need them there either...